pkg/template/api/types.go

```diff
@@ -13,6 +13,9 @@ type Template struct {
 	unversioned.TypeMeta
 	kapi.ObjectMeta
 
+	// Optional: An instructional message for the user processing this template (parameter substitution will be performed).
+	PostProcessMessage string
```

Comment:
I don't see anyone using lower-camel-case for the type definitions. I've committed with upper-camel-case to see if you agree. It seems like the convention is less formal here since it is surprisingly difficult to start the sentence with the noun and maintain natural language without repetition.